contract/src/test/java/io/neow3j/contract/TransactionBuilderTest.java

```diff
@@ -568,51 +669,47 @@ public void callWithoutParams() throws IOException {
         );
         Wallet w = Wallet.withAccounts(account1);
 
```

Comment:
This test method doesn't belong here anymore. It tests the behavior of NeoToken or SmartContract but not TransactionBuilder.

Reply:
True, actually it already exists in SmartContractTest, so I'll remove this test.